narayana-spring-boot-starter-it/src/test/resources/application.properties

```diff
@@ -1,5 +1,6 @@
 logging.level.org.apache.activemq.audit.base=ERROR
 logging.level.org.apache.activemq.audit.message=ERROR
+logging.level.org.apache.activemq.audit.resource=ERROR
```

Comment:
@graben please can you say why this line was added?

Reply:
It's one of three log levels of Artemis which should be ERROR to avoid massive logging entries.

Reply:
Done